### DISCUSSION TITLE
passing binarized image to ocr_predictor.

### DISCUSSION THREAD

**Discussion by bharathchandra0915** [mindee/doctr#1470] [Q&A]
I'm not sure whether pre-processing is done before the text recognition is performed. Hence, I had done my own pre-processsing steps according to my use case. So, now I need to pass a binarized image to the ocr_predictor model. But then I'm getting the following error:

    raise ValueError("incorrect input shape: all pages are expected to be multi-channel 2D images.")
ValueError: incorrect input shape: all pages are expected to be multi-channel 2D images.

Please help me, on how to pass binarized image as input to model.
Or, is pre-processing really important? does mindee doctor pre-process image before text recognition!?

Thank you

**Comment by felixdittrich92** [comment]
Hi @bharathchandra0915 :wave:,

Normally you don't need any pre-processing (pad and resize to 1024x1024 is done internally).
So the input should be RGB or grayscale (1 channel)
Could you describe your specific use case a bit more in detail ? 

Best regards,
Felix :)
